<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>995</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>995</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>995</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>599</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/wallpaper1.jpg);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1898</width>
        <height>717</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>1898</width>
        <height>717</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(7, 7, 7);
font: 75 14pt &quot;Tw Cen MT&quot;;</string>
      </property>
      <property name="text">
       <string>..or drop here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="toNeuralButton">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(13, 183, 168, 168), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(13, 183, 168, 168));

</string>
      </property>
      <property name="text">
       <string>FIND DROPS</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ImageProcessingButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 183, 168);
font:  26pt &quot;Tw Cen MT&quot;;
color: rgb(27, 27, 27);</string>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="BackButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 183, 168);
font:  26pt &quot;Tw Cen MT&quot;;
color: rgb(27, 27, 27);</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="backgr.qrc"/>
 </resources>
 <connections/>
</ui>
